javaagent-tooling/src/test/groovy/io/opentelemetry/javaagent/extension/instrumentation/InstrumentationModuleTest.groovy

```diff
@@ -77,18 +77,12 @@ class InstrumentationModuleTest extends Specification {
 
   def "configure default sys prop as #value"() {
     setup:
-    Config.INSTANCE = new ConfigBuilder().readProperties([
+    Config config = new ConfigBuilder().readProperties([
       "otel.instrumentation.common.default-enabled": String.valueOf(value)
     ]).build()
-    def target = new TestInstrumentationModule(["test"])
-    target.extend(new AgentBuilder.Default())
 
     expect:
-    target.enabled == enabled
-    target.applyCalled == enabled
-
-    cleanup:
-    Config.INSTANCE = null
+    InstrumentationModule.initDefaultEnabled(config) == enabled
```

Comment:
Is this method called only from test? They I don't see any point in neither the method nor the test.